service/controller/cluster.go

```diff
@@ -180,6 +180,11 @@ func newClusterResources(config ClusterConfig) ([]resource.Interface, error) {
 			StateGetter: appGetter,
 		}
 
+		c.AllowedAnnotations = []string{
+			"app-operator.giantswarm.io/latest-configmap-version",
```

Comment:
It could be flags value but I don't want to make cluster-operator overcomplicated toward managed app environment. 
So let's keep these annotations in here, WDYT?

Reply:
Let's add them to apiextensions since we need them in multiple operators now.
https://github.com/giantswarm/apiextensions/tree/master/pkg/annotation

They are already in the app library so then we can remove them there there. https://github.com/giantswarm/app/blob/0bb7144669b2189ec280c6e2d6859f955dbc119b/pkg/annotation/annotation.go#L22